shared/src/business/useCases/trialSessions/updateTrialSessionInteractor.js

```diff
@@ -188,31 +175,35 @@ exports.updateTrialSessionInteractor = async (
       }
     }
 
-    const serviceInfo = await applicationContext
+    serviceInfo = await applicationContext
       .getUseCaseHelpers()
       .savePaperServicePdf({
         applicationContext,
         document: paperServicePdfsCombined,
       });
     pdfUrl = serviceInfo.url;
+  }
 
-    await applicationContext.getNotificationGateway().sendNotificationToUser({
-      applicationContext,
-      message: {
-        action: 'notice_generation_complete',
-        docketEntryId: serviceInfo.docketEntryId,
-        hasPaper: serviceInfo.hasPaper,
-        pdfUrl,
-      },
-      userId: user.userId,
-    });
+  if (trialSession.swingSession && trialSession.swingSessionId) {
+    newTrialSessionEntity.setAsSwingSession(trialSession.swingSessionId);
   }
 
   await applicationContext.getPersistenceGateway().updateTrialSession({
     applicationContext,
     trialSessionToUpdate: newTrialSessionEntity.validate().toRawObject(),
   });
 
+  await applicationContext.getNotificationGateway().sendNotificationToUser({
+    applicationContext,
+    message: {
+      action: 'update_trial_session_complete',
+      hasPaper: serviceInfo?.hasPaper,
+      pdfUrl,
```

Comment:
Seems like this logic was moved from `updateTrialSessionAction` to this interactor, if it's staying can test coverage be added for this new logic?